design/one-pager-stacks-security-isolation.md

```diff
@@ -0,0 +1,187 @@
+# Stacks Security and Isolation
+
+* Owner: Jared Watts (@jbw976)
+* Reviewers: Crossplane Maintainers
+* Status: Draft, revision 1.0
+
+## Background
+
+The current model for extending Crossplane with new functionality is scoped to a single type that is used in all scenarios and by all personas.
+All new types of functionality, from an end user wanting to install a simple application to an administrator wanting to install support for low level infrastructure types, are all treated the same.
+They have the same approach to security and access control and the artifacts that they install have the same scope of impact to the control plane.
+
+This one size fits all approach is not ideal though, the multiple scenarios and personas need finer grain control around security, access, and isolation.
+This design doc proposes a new approach for the Crossplane Stack Manager (SM) that will support this increased control for the installation of stacks.
+
+## Personas and Scenarios
+
+There are many different scenarios and types of users that may want to install new functionality into their control plane.
+This design proposes support for two high level personas/scenarios as an effective starting point.
+
+* **Platform:** An administrator of the control plane that is installing wide ranging support for lower level infrastructure and platform components,
+such as installing support for an entire cloud provider and all of its infrastructure and platform services.
+  * An example of this would be the `GCP` stack that contains a `Network` primitive and the `CloudSQL` resource.
```

Comment:
Why is `Network` a "primitive" and `CloudSQL` a "resource"? Presuming we mean `CloudSQLInstance` here, they're both managed resources.